modules/core/shared/src/main/scala/lucuma/core/math/Place.scala

```diff
@@ -3,47 +3,59 @@
 
 package lucuma.core.math
 
+import cats.implicits._
 import cats.Eq
 import cats.Show
+import java.time.ZoneId
 import coulomb.Quantity
 import coulomb.si.Meter
 import eu.timepit.refined.types.numeric.NonNegInt
-import monocle.Lens
-import monocle.macros.GenLens
-import java.time.ZoneId
+import monocle.Getter
+import io.chrisdavenport.cats.time._
+import coulomb.cats.implicits._
+import eu.timepit.refined.cats._
 
 /** A point on Earth, given latitude, longitude and altitude in m above sea level. */
-final case class Place(
-  latitude:  Lat,
-  longitude: Lon,
-  altitude:  Quantity[NonNegInt, Meter],
-  zoneId:    ZoneId
+class Place(
```

Comment:
Should this maybe go back to being a `final case class` since we're not trying to extend it anymore?

Reply:
Yup, good catch. It can be a final case class again and we don't need constructor and can derive Eq and Show.

Reply:
:+1: